KomodoPlatform/komodo-wallet-desktop
# Fix/bestorders overflow

Fixes reported overflow in current RC. 

To test:
- Go to simple dex view
- Select MARTY, then open the buy coin selection list
- Confirm no overflow from tickers or values in columns.
